packages/common/src/bls/ERC20CalldataCacheDecoderInteractor.ts

```diff
@@ -97,11 +97,17 @@ export class ERC20CalldataCacheDecoderInteractor implements ICalldataCacheDecode
   }
 
   async compressAddressesToIds (addresses: Address[][]): Promise<AddressesCachingResult> {
-    return { ids: addresses.map(it => it.map(toBN)), writeSlotsCount: 0 }
+    return {
```

Comment:
again here, for ERC20 the addresses are just the holders of the token, a single array